### ISSUE TITLE
Lettuce 6.4.0.M1 was picked over 6.4.0.RELEASE

### ISSUE THREAD

**Issue by danielcompton** [liquidz/antq#256]
It's a bit of a strange versioning scheme, but we had this with a recent [Lettuce release](https://github.com/redis/lettuce/releases):

```clj
io.lettuce/lettuce-core {:mvn/version "6.3.2.RELEASE"}
```

and antq upgraded it to

```clj
io.lettuce/lettuce-core {:mvn/version "6.4.0.M1"}
```

when it should have picked

```clj
io.lettuce/lettuce-core {:mvn/version "6.4.0.RELEASE"}
```

**Comment by liquidz** [comment]
@danielcompton Thanks for your reporting!
It seems a problem in [xsc/version-clj](https://github.com/xsc/version-clj).
```clojure
(version-clj.core/version-compare "6.4.0.M1" "6.4.0.RELEASE") ;; => 1
```

I'll have a look and report to version-clj.

**Comment by liquidz** [comment]
cf. https://github.com/xsc/version-clj/pull/21

**Comment by liquidz** [comment]
@danielcompton Just released v2.9.1221 :)

**Comment by danielcompton** [comment]
Lovely, thank you @liquidz!
